cloud-formation/contributions-app.yml

```diff
@@ -121,10 +121,14 @@
                       - " s3 cp s3://contributions-private/"
                       -
                         Ref: "Stage"
-                      - "/giraffe-keys.conf /etc/gu"
+                      - "/contributions.private.conf /etc/gu"
+
+
                 - "wget https://s3.amazonaws.com/aws-cloudwatch/downloads/latest/awslogs-agent-setup.py"
                 - "export CONF_DIR=/etc/contributions-frontend"
-                - "dpkg -i /tmp/contributions-frontend_1.0-SNAPSHfix OT_all.deb"
```

Comment:
This was accidentally added with https://github.com/guardian/contributions-frontend/commit/f722cfa :-)